api/types/access_request.go

```diff
@@ -401,6 +401,11 @@ func (r *AccessRequestV3) SetLoginHint(login string) {
 	r.Spec.LoginHint = login
 }
 
+// SetAssumedRoleRequestID sets the request ID of the access request from which the assumed role was obtained
+func (m *Metadata) SetAssumedRoleRequestID(requestId string) {
```

Comment:
```suggestion
func (m *Metadata) SetAssumedRoleRequestID(requestID string) {
```

In go, convention is for initialisms to have the same case. (Yes, this conflicts with our JS convention..)